pkg/services/ngalert/notifier/channels/util.go

```diff
@@ -82,14 +82,16 @@ func getImage(ctx context.Context, l log.Logger, imageStore ImageStore, alert ty
 // images have been found.
 func withStoredImages(ctx context.Context, l log.Logger, imageStore ImageStore, forEachFunc forEachImageFunc, alerts ...*types.Alert) error {
 	for index, alert := range alerts {
-		img, err := getImage(ctx, l, imageStore, *alert)
+		logger := l.New("alert", alert.String())
```

Comment:
This struct has an overridden `String()` that is log-friendly. This doesn't log a ton of extra details. It visually matches the existing alertmanager logs, it'll look like `alert=SomeRule[bc0d64]`